internal/controller/controllers/infraenv_controller.go

```diff
@@ -345,14 +345,15 @@ func (r *InfraEnvReconciler) ensureISO(ctx context.Context, log logrus.FieldLogg
 }
 
 func CreateInfraEnvParams(infraEnv *aiv1beta1.InfraEnv, imageType models.ImageType, pullSecret string, clusterID *strfmt.UUID, openshiftVersion string) installer.RegisterInfraEnvParams {
+	normalizedRHCOSARCH := infraEnv.Spec.CpuArchitecture
```

Comment:
Maybe too quick and missed some other logic, but don't you want to write here `... := NormalizeRHCOSArch(infraEnv.Spec.CpuArchitecture)`?

Reply:
I meant to remove this all together, but will add `NormalizeRHCOSArch(infraEnv.Spec.CpuArchitecture)` back based on our slack conversation.